swarm/swarm.go

```diff
@@ -178,6 +178,21 @@ func NewSwarm(ctx *node.ServiceContext, backend chequebook.Backend, config *api.
 	if ensresolver == nil {
 		log.Warn("No ENS API specified, resource updates will NOT validate resource update chunks")
 	}
+	// setup local store
+	log.Debug(fmt.Sprintf("Set up local storage"))
```

Comment:
No need for `Sprintf`.

As a side note, we have structured logs, so generally we should avoid using `Sprintf` in `log.Debug`, `Trace`, `Error`, etc.